lib/OrdinaryDiffEqFIRK/src/algorithms.jl

```diff
@@ -1,20 +1,41 @@
-# FIRK Methods
 
-"""
-@article{hairer1999stiff,
-title={Stiff differential equations solved by Radau methods},
-author={Hairer, Ernst and Wanner, Gerhard},
-journal={Journal of Computational and Applied Mathematics},
-volume={111},
-number={1-2},
-pages={93--111},
-year={1999},
-publisher={Elsevier}
-}
+hairer1999stiff = """@article{hairer1999stiff,
+    title={Stiff differential equations solved by Radau methods},
+    author={Hairer, Ernst and Wanner, Gerhard},
+    journal={Journal of Computational and Applied Mathematics},
+    volume={111},
+    number={1-2},
+    pages={93--111},
+    year={1999},
+    publisher={Elsevier}}"""
 
-RadauIIA3: Fully-Implicit Runge-Kutta Method
-An A-B-L stable fully implicit Runge-Kutta method with internal tableau complex basis transform for efficiency.
-"""
+extra_keyword_description = """
+    - `extrapolant`: TBD
+    - `fast_convergence_cutoff`: TBD
+    - `new_W_γdt_cutoff`: TBD
+    - `controller`: TBD
+    - `κ`: TBD
+    - `maxiters`: TBD
+    - `smooth_est`: TBD
+    - `step_limiter!`: TBD"""
+extra_keyword_default = """
+    extrapolant = :dense,
+    fast_convergence_cutoff = 1 // 5,
+    new_W_γdt_cutoff = 1 // 5,
+    controller = :Predictive,
```

Comment:
@ChrisRackauckas 
For FIRK methods, this page only implements Predictive controller.
https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/lib/OrdinaryDiffEqFIRK/src/controllers.jl
Am I correct to assume that any other controller, such as PI, will thus not work?

Also, I don't see these methods in the `contollers.jl` file used anywhere else in `OrdinaryDiffEqFIRK`?